### PR TITLE
Fix Linux build: Make the `nasm` feature a no-op

### PR DESCRIPTION
**Pull request by emilk**
### What
* Circumvent https://github.com/rerun-io/rerun/issues/7671

### Checklist
* [x] I have read and agree to [Contributor Guide](https://github.com/rerun-io/rerun/blob/main/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/rerun-io/rerun/blob/main/CODE_OF_CONDUCT.md)
* [x] I've included a screenshot or gif (if applicable)
* [x] I have tested the web demo (if applicable):
  * Using examples from latest `main` build: [rerun.io/viewer](https://rerun.io/viewer/pr/7675?manifest_url=https://app.rerun.io/version/main/examples_manifest.json)
  * Using full set of examples from `nightly` build: [rerun.io/viewer](https://rerun.io/viewer/pr/7675?manifest_url=https://app.rerun.io/version/nightly/examples_manifest.json)
* [x] The PR title and labels are set such as to maximize their usefulness for the next release's CHANGELOG
* [x] If applicable, add a new check to the [release checklist](https://github.com/rerun-io/rerun/blob/main/tests/python/release_checklist)!
* [x] If have noted any breaking changes to the log API in `CHANGELOG.md` and the migration guide

- [PR Build Summary](https://build.rerun.io/pr/7675)
- [Recent benchmark results](https://build.rerun.io/graphs/crates.html)
- [Wasm size tracking](https://build.rerun.io/graphs/sizes.html)

To run all checks from `main`, comment on the PR with `@rerun-bot full-check`.

### CODE REVIEW

**Comment by emilk** [comment]
@rerun-bot full-check

File: crates/store/re_video/Cargo.toml [hunk 1]
```diff
@@ -31,6 +31,7 @@ av1 = ["dep:dav1d"]
 
 ## Enable faster native video decoding with assembly.
 ## You need to install [nasm](https://nasm.us/) to compile with this feature.
+## TODO(#7671): there is a bug that makes this incompatible with arch linux
```
Comment (jprochazk) [inline crates/store/re_video/Cargo.toml:34]:
```suggestion
## TODO(#7671): there is a bug that makes this incompatible with linux
```

File: crates/top/rerun-cli/Cargo.toml [hunk 2]
```diff
@@ -43,6 +43,7 @@ default = ["native_viewer", "web_viewer"]
 
 ## Enable faster native video decoding with assembly.
 ## You need to install [nasm](https://nasm.us/) to compile with this feature.
+## TODO(#7671): there is a bug that makes this incompatible with arch linux
```
Comment (jprochazk) [inline crates/top/rerun-cli/Cargo.toml:46]:
```suggestion
## TODO(#7671): there is a bug that makes this incompatible with linux
```

File: crates/top/rerun/Cargo.toml [hunk 3]
```diff
@@ -73,6 +73,7 @@ log = ["dep:env_logger", "dep:log"]
 
 ## Enable faster native video decoding with assembly.
 ## You need to install [nasm](https://nasm.us/) to compile with this feature.
+## TODO(#7671): there is a bug that makes this incompatible with arch linux
```
Comment (jprochazk) [inline crates/top/rerun/Cargo.toml:76]:
```suggestion
## TODO(#7671): there is a bug that makes this incompatible with linux
```

File: rerun_py/Cargo.toml [hunk 4]
```diff
@@ -30,6 +30,7 @@ extension-module = ["pyo3/extension-module"]
 
 ## Enable faster native video decoding with assembly.
 ## You need to install [nasm](https://nasm.us/) to compile with this feature.
+## TODO(#7671): there is a bug that makes this incompatible with arch linux
```
Comment (jprochazk) [inline rerun_py/Cargo.toml:33]:
```suggestion
## TODO(#7671): there is a bug that makes this incompatible with linux
```

**Review by jprochazk — APPROVED** [review]

**Comment by emilk** [comment]
`main` build still fails because of `--all-features`… maybe I'll do a different fix

**Comment by emilk** [comment]
@rerun-bot full-check

**Pull request merged**